app/services/migration/migrators/application.rb

```diff
@@ -71,6 +71,13 @@ def call
 
         if ecf_npq_application.school_urn.present?
           application.school_id = find_school_id!(urn: ecf_npq_application.school_urn)
+
+          if application.school.ukprn.to_s == ecf_npq_application.school_ukprn.to_s
+            application.ukprn = ecf_npq_application.school_ukprn
```

Comment:
since we're setting this here can we delete this line https://github.com/DFE-Digital/npq-registration/pull/1832/files#diff-a44bc9ad55c1f2d7565d5a1a513776fb08f5180b4934350ee83fba59bad1d52cR94